satellite/api.go

```diff
@@ -205,14 +205,12 @@ func NewAPI(log *zap.Logger, full *identity.FullIdentity, db DB,
 	var err error
 
 	{
-		if !versionInfo.IsZero() {
```

Comment:
Why did we remove this check?

Reply:
i just think we should always log it

Reply:
I am not sure if this will fail for development builds. I would expect a nil pointer

Reply:
this is what it prints: https://github.com/storj/private/blob/master/version/version.go#L43
 so i think it will just print zero values like this:
```
satellite-garbage-collection/0            11:44:02.347 | INFO	satellite/gc.go:106	Version info	{"Version": "0.0.0", "Commit Hash": "", "Build Timestamp": "0001-01-01 00:00:00 +0000 UTC", "Release Build": false}
```

Reply:
Okay, great. we print default values currently.